SECEdgar/extractor/EDGARExtractor.py

```diff
@@ -0,0 +1,227 @@
+import re
+import json
+import os.path
+import logging
+import uu
+
+
+class EDGARExtractor():
```

Comment:
Can we name this class as `FilingsExtractor` because as per your description it handles extracting filings from a single file.

Reply:
ok